### PR TITLE
add all used gazebo model files into repository so we don't need to f…

### PR DESCRIPTION
**Pull request by le-horizon**
…etch them from gazebosim.org, and we can maintain stability of these models.

These are all the models I can find from the current repo.
We probably need to check it every time we add a new model to SocialRobot.

Thanks,
Le

### CODE REVIEW

**Comment by le-horizon** [comment]
> Please add some description about the source of these files
> Add make sure to we follow the license requirement: https://bitbucket.org/osrf/gazebo_models/src/default/LICENSE
> 
> Is it possible to disable downloading from the internet?

Good points as always @emailweixu.  Done both.  PTAL?

Thanks,
Le

File: python/social_bot/models/database.config [hunk 1]
```diff
@@ -1,13 +1,26 @@
 <?xml version='1.0'?>
 <database>
-  <name>Models used by SocialBot</name>
+  <name>Models used by SocialBot, copied or adapted from https://bitbucket.org/osrf/gazebo_models</name>
```
Comment (emailweixu) [inline python/social_bot/models/database.config]: not all of them are from https://bitbucket.org/osrf/gazebo_models.
 ball, texture_ground_plane, cartpole are created by us
 pr2 and youbot_noplugin are from somewhere else

Comment (Jialn) [reply python/social_bot/models/database.config]: pr2 and yobot are also copied from there...  iCub is from https://github.com/robotology/icub-gazebo

**Comment by Jialn** [comment]
Thanks for the work. It's more convient now. There's problem download these models in mainland China I have to copy model file by hand everytime setup the running environment.

**Comment by le-horizon** [comment]
Glad to know it's helpful, Jiangtao.  I updated the comments, let me know if you want to make any changes to where the model files are from.

Thanks,
Le

**Pull request merged**